examples/form-to-notion-db/.env

```diff
@@ -0,0 +1,2 @@
+NOTION_TOKEN=<notion-integration-key>
```

Comment:
Let's rename this file to `.env.example` to keep it consistent with other examples

Reply:
done!